modules/tinymce/src/themes/silver/test/ts/headless/components/urlinput/UrlInputTest.ts

```diff
@@ -75,7 +75,7 @@ describe('headless.tinymce.themes.silver.components.urlinput.UrlInputTest', () =
   const closeMenu = () => {
     const doc = hook.root();
     // Close the menu
-    Keyboard.activeKeydown(doc, Keys.escape());
+    Keyboard.activeKeystroke(doc, Keys.escape());
```

Comment:
This stopped working when @HAFRMO changed alloy to listen to `keyup` for the escape key. As such the tests were flaky and only worked because we have retry enabled so this corrects that.